packages/vite-config/src/baseConfig.ts

```diff
@@ -17,7 +18,7 @@ const baseConfig: UserConfig = defineConfig({
       transformMixedEsModules: true,
     },
   },
-  plugins: [nodePolyfills()],
+  plugins: [nodePolyfills(), tsconfigPaths()],
```

Comment:
The vite-tsconfig-paths plugin inspects the tsconfig `paths` property and modifies the Vite config so modules are resolved in the same way TypeScript resolves types, ensuring our type-checking and bundling are consistent.